Mapping/Loader/AttributesLoader.php

```diff
@@ -54,30 +55,93 @@ public function __construct(
      * {@inheritdoc}
      */
     public function loadClassMetadata(
-        ClassMetadata $classMetadata,
+        ClassMetadataInterface $classMetadata,
         array $normalizationGroups = null,
         array $denormalizationGroups = null,
         array $validationGroups = null
     ) {
+        $this->populateFromSerializerMetadata($classMetadata, $normalizationGroups, $denormalizationGroups);
```

Comment:
Instead of having all these logic in the attributes loader (I'm speaking about populate from serializer, setters, issers, getters and hassers) I would say that it may be more interesting to use a chain of responsibility here and chain the different `AttributeLoaders`, don't you think ?